.github/workflows/check_documentation_build.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
         git config --local user.email "opensource@exasol.com"
         git config --local user.name "GitHub Action"
         git fetch
-        poetry run python -m nox -s push-pages-current
+        poetry run python -m nox -v -s push-pages-current
```

Comment:
use long form for  `-v` if available, could find the doc for this parameter

Reply:
@tkilias long option definitely would make sense, it's a left over though.
Good catch in any case.

```
  -v, --verbose         Logs the output of all commands run including commands marked silent.
```